server/views/layouts/publicLayout.handlebars

```diff
@@ -32,6 +32,7 @@
   {{/extend}}
 
   {{{render 'head-scripts'}}}
+  {{#if cookies}}
```

Comment:
Because it's specifically analytics consent, it is better to rename `cookies` to `klaroAnalyticsConsentCookie` to keep it consistent. (In case in future we will need other type of consent cookie.)